client-v2/src/shared/components/article/ArticleBody.tsx

```diff
@@ -342,7 +342,7 @@ const articleBodyDefault = React.memo(
                   {cutoutThumbnail ? (
                     <ThumbnailCutout src={cutoutThumbnail} />
                   ) : null}
-                  {hasMainVideo && (
+                  {showMainVideo && (
```

Comment:
Are we certain this is the behaviour we want? We already have an indicator for `showMainVideo` on line 355. I think this icon is there to reveal that video is available, not that it's been overridden -- but I might be wrong!

Reply:
If the purpose of the icon is to show that here is a video _available_, rather than that the video is toggled on, then we should definitely preserve the existing behaviour.